hw/top_earlgrey/data/ip/chip_otbn_testplan.hjson

```diff
@@ -157,5 +157,107 @@
         "OTBN.KEYMGR",
       ]
     }
+    {
+      name: chip_sw_otbn_rsa
+      desc: '''Check that the OTBN can run RSA.
+
+            Check that OTBN can run RSA to:
+             - Encrypt data.
+             - Decrypt data.
+             - Sign data.
+             - Verify sined data.
+            With the keys sizes 2048, 3072 and 4096 bits.
+      '''
+      stage: V3
```

Comment:
Is it possible to mar these test cases for SiVal only? I am not sure we can support them in DV given the test run time.

Reply:
The same applies to other test cases here. I recommend to tracking cryptolib test cases in a separate testplan al together. Similar to how we did with the ROM e2e testplan.

Reply:
Agreed, the RSA tests (especially keygen below) take quite a long time with the larger keys -- to be more precise, ~tens of millions of cycles for signing or decryption  (the `*_signature_functest` targets do sign-then-verify, and the `*_encryption_functest` ones do encrypt-then-decrypt) , and keygen can easily go to hundreds of millions of cycles.

Reply:
Moved the new testpoints to `sw/device/lib/crypto/data/crypto_testplan.hjson` and changed `stage` to `NA`.